cmd/compose-controller/main.go

```diff
@@ -153,7 +159,9 @@ func start(opts *controllerOptions) error {
 	log.Infof("Controller ready")
 
 	if opts.healthzCheckPort > 0 {
-		healthz.DefaultHealthz()
```

Comment:
Note to reviews, this function was removed.

Reply:
This needs to be tested: the healthz endpoint must accept plain http requests (not https), and return 200-299 status codes.
The default mux, if not added with an handler returns a 404 status code, I think

Reply:
This is a straight copy of the code from [`DefaultHealthz()`](https://github.com/docker/compose-on-kubernetes/pull/146/files#diff-f28b48f780d2d8653a4feffbf1ab3b33L40-L48) so the behaviour should be the same.

I ran the following test:
1. Install Compose on Kubernetes with healthz port specified
1. Run a pod in the same namespace and curl the endpoint:

```console
curl http://10.1.0.32:8080/healthz
ok
curl https://10.1.0.32:8080/healthz
curl: (35) error:1408F10B:SSL routines:ssl3_get_record:wrong version number
```